src/Microsoft.Health.Fhir.CosmosDb.Core/Features/Storage/StoreProcedures/IStoredProcedureMetadata.cs

```diff
@@ -2,12 +2,14 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
 // -------------------------------------------------------------------------------------------------
-
+using System.Linq;
+using System.Text;
```

Comment:
I believe the using statements can be simplified in this class.

Reply:
done